source/customizations/main-navigation.inc

```diff
@@ -1,205 +1,372 @@
 
-Navigation links can be used as top level navigation items or as links within a navigation menu.
+Menu items are automatically grouped based on the applications ``subcategory``.
+Use the ``apps`` array to define the app tokens for this navigation menu.
 
-* **Static links:** these are pre-defined links ready to use with a simple string definition.
-  Currently supported static links are: ``all_apps``, ``sessions``, ``support_ticket``, ``docs``, ``products_dev``, ``products_usr``, ``log_out``, and ``restart``.
+The order of the menu items is based on the order you've defined.
 
 .. code-block:: yaml
 
   nav_bar:
-    # Navigation link
-    - title: "Menu Item"
+    - title: "My Apps"
+      icon: "fas://window-restore"
+      apps:
+        - "sys/Rstudio/*"
+        - "sys/bc_desktop"
+
+.. figure:: /images/def-apps-menu.png
+    :align: center
+    :alt: An open dropdown menu titled "My Apps". The menu shows two submenus. The first is titled "Servers" and has five list items for all the discovered through the "sys/Rstudio/*" configuration. The second submenu is titled "Desktops" and has one list item for the discovered through the "sys/bc_desktop" configuration.
+
+
+.. _menus-based-on-links:
+
+Menus based on links
+********************
+
+Menus based on links provide even more granular control on how the dropdown menu is generated.
+
+As seen in the previous sections, using an ``apps`` array will automatically generate
+submenus based on the ``subcategory`` of the applications that have been discovered.
+
+When using links, we can override this behaviour to provide different titles to
+the submenus, or with no submenus and subtitles at all.
+
+Let's look at this example using the same application tokens, but using ``links`` instead
+of ``apps``. The system will discover and generate list items for the same eight applications,
+only without **any** submenus.  It is simply a list of all the discovered applications with
+no further grouping.
+
+.. code-block:: yaml
+
+  nav_bar:
+    - title: "My Apps"
+      icon: "fa://code"
       links:
-        - "all_apps"
-        - "sessions"
-        - "support_ticket"
-        - "docs"
-        - "products_dev"
-        - "products_usr"
-        - "log_out"
-        - "restart"
-
-.. figure:: /images/def-static-links.png
-   :align: center
-
-* **App links:** applications identified by the tokens are added to the menu.
-  When configured as a top level link, only one application token is supported.
-  If the token matches multiple applications, the navigation entry is ignored.
-
-  When configured as a ``links`` array item, the subcategory will not be used to group the matched applications.
-  The definition order is respected when rendering the links.
-  If grouping is required, use the ``group`` property to add the desired structure as shown in the
-  :ref:`applications based menu above. <applications_based_menu>`
+        - "sys/Rstudio/*"
+        - "sys/bc_desktop"
+
+
+**IMAGE NEEDED**
```

Comment:
I need an image here for this config

```yaml

  nav_bar:
    - title: "My Apps"
      icon: "fa://code"
      links:
      links:
        - "sys/Rstudio/*"
        - "sys/bc_desktop"
```

Reply:
Sure - Here it is:

<img width="1067" alt="def-menu-links" src="https://user-images.githubusercontent.com/17336210/219465659-6623fdae-dcee-4654-90e2-d4c52b275b18.png">